### DISCUSSION TITLE
Renovate does not work with Kustomization

### DISCUSSION THREAD

**Discussion by pulingupta** [renovatebot/renovate#19987] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

My Renovate.json look like below
"{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "kubernetes": {
    "fileMatch": ["base/.+\\.yaml$"]
  }
}"
but it does not match kustomization.yaml and so no Pr is raised, please help.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
Wrong config. needs to be `kustomize`

https://docs.renovatebot.com/modules/manager/kustomize/
